### PR TITLE
Add Resource for V2 SCC Findings Exports to Big Query Organization Config

### PR DESCRIPTION
**Pull request by vijaykanthm**
Related to http://b/356159589.

Bug Description: https://github.com/hashicorp/terraform-provider-google/issues/18848

Adding new template for V2 SCC Findings Exports to Big Query Organization Config
https://cloud.google.com/security-command-center/docs/reference/rest/v2/organizations.locations.bigQueryExports

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_scc_v2_organization_scc_big_query_exports`
```

### CODE REVIEW

**Review by hao-nan-li — COMMENTED** [review]
Looks like there's some lint issue and vcr test failures. Could you take a look into them?

File: mmv1/products/securitycenterv2/OrganizationSccBigQueryExports.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs/how-to-export-data'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/organizations.locations.bigQueryExports'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_organization_big_query_export_config_basic'
+    primary_resource_id: 'custom_big_query_export_config'
+    external_providers: ["random", "time"]
+    skip_test: true
```
Comment (hao-nan-li) [inline mmv1/products/securitycenterv2/OrganizationSccBigQueryExports.yaml:41]: I noticed you fixed the previous error from the VCR test, should this test still be skipped?

Comment (vijaykanthm) [reply mmv1/products/securitycenterv2/OrganizationSccBigQueryExports.yaml:41]: I can remove it and see if it works as expected.

**Comment by hao-nan-li** [comment]
> Tests failed during RECORDING mode: `TestAccSecurityCenterV2OrganizationSccBigQueryExports_sccV2OrganizationBigQueryExportConfigBasicExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-11428/artifacts/c9a6237f-6bd3-4a8c-b46c-fab97deff18b/build-log/recording_build/TestAccSecurityCenterV2OrganizationSccBigQueryExports_sccV2OrganizationBigQueryExportConfigBasicExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-11428/artifacts/c9a6237f-6bd3-4a8c-b46c-fab97deff18b/recording/TestAccSecurityCenterV2OrganizationSccBigQueryExports_sccV2OrganizationBigQueryExportConfigBasicExample.log)]
> 
> Errors occurred during RECORDING mode. Please fix them to complete your PR.
> 
> View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-11428/artifacts/c9a6237f-6bd3-4a8c-b46c-fab97deff18b/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-11428/artifacts/c9a6237f-6bd3-4a8c-b46c-fab97deff18b/recording) for each test

Looks like it failed again, since it passed locally, could you attach the log and disable this test? I think this should be good to go.

**Review by hao-nan-li — APPROVED** [review]

**Pull request merged**